test/TestBase.py

```diff
@@ -237,7 +237,8 @@ def testAllStateCanReachEverything(self):
             for location in self.multiworld.get_locations():
                 if location.name not in excluded:
                     with self.subTest("Location should be reached", location=location):
-                        self.assertTrue(location.can_reach(state), f"{location.name} unreachable")
+                        reachable = location.can_reach(state)
```

Comment:
why chane this?

Reply:
Ease of debugging to be honest. I did not remember doing it, but I do believe it's an improvement, although an extremely minor one. I'm not willing to die on this hill regardless so if you want, I'll revert it.